setup.py

```diff
@@ -122,7 +122,7 @@
 package_info['package_data'][PACKAGENAME].extend(c_files)
 
 # I'm not sure whether it makes sense to duplicate this here or not
-default_install_requires = 'astropy asdf pytest==3.1'
+default_install_requires = 'astropy asdf pytest>=3.1'
```

Comment:
Regarding the comment above, maybe the answer is 'no' and we could simply do ``metadata['install_requires']`` below?

Reply:
I'm not opposed in principle, but right now every other field that we get from the `metadata` dict has a default value. Should we git rid of all of those as well?